sender/influxdb.go

```diff
@@ -146,21 +147,14 @@ func (idb *InfluxDB) Send(c *skogul.Container) error {
 		}
 		fmt.Fprintf(&buffer, "%s", measurement)
 		for key, value := range m.Metadata {
-			var field interface{}
-			v, ok := value.(string)
-			if ok {
-				field = idb.replacer.Replace(v)
-			} else {
-				field = value
-			}
-			fmt.Fprintf(&buffer, ",%s=%v", idb.replacer.Replace(key), field)
+			fmt.Fprintf(&buffer, ",%s=%v", idb.replacer.Replace(key), idb.writeValue(value))
 			nmdata++
 		}
 		fmt.Fprintf(&buffer, " ")
 		comma := ""
 		for key, value := range m.Data {
 
-			fmt.Fprintf(&buffer, "%s%s=%#v", comma, idb.replacer.Replace(key), value)
+			fmt.Fprintf(&buffer, "%s%s=%#v", comma, idb.replacer.Replace(key), idb.writeValue(value))
```

Comment:
This actually breaks string handling.

InfluxDB differentiates between how tag values and field values should be treated, so only tag values need to use idb.replacer.Replace() on the value, while field values are handled by double quoting, which is why one format is %v and the other %#v (I wonder what they were smoking...).

See https://docs.influxdata.com/influxdb/v1.7/write_protocols/line_protocol_reference/#quoting